src/redux/testStore.js

```diff
@@ -0,0 +1,14 @@
+import { applyMiddleware, combineReducers, createStore } from 'redux';
+import thunk from 'redux-thunk';
+import missionsReducer from './missions/missions';
+import rocketsReducer from './rockets/rockets';
+
+const testStore = createStore(
+  combineReducers({
+    missionsReducer,
+    rocketsReducer,
+  }),
+  applyMiddleware(thunk),
+);
+
+export default testStore;
```

Comment:
- [ ] OPTIONAL The "nontest" store and this test store are the same!! The only difference is the logger module, you should try to find a way of refactoring the original or just use the original instead of making a new file!